lib/internal/modules/esm/fetch_module.js

```diff
@@ -39,14 +39,16 @@ const path = require('path');
  */
 const cacheForGET = new SafeMap();
 
+// [1] The V8 snapshot doesn't like some C++ APIs to be loaded eagerly. Do it
+// lazily/at runtime and not top level of an internal module.
+
+// [2] Creating a new agent instead of using the gloabl agent improves
+// performance and precludes the agent becoming tainted.
```

Comment:
```suggestion
// [2] Creating a new agent instead of using the global agent improves
// performance and precludes the agent from becoming tainted.
```

Reply:
Oh! sorry, I thought I committed your suggestion (via GitHub). Thanks for pushing it!